CMakeLists.txt

```diff
@@ -6,8 +6,13 @@ if(${CMAKE_VERSION} STREQUAL "2.8.2")
 endif()
 
 # This can be read from ${PROJECT_NAME} after project() is called
-project(Minetest)
-string(TOLOWER ${PROJECT_NAME} PROJECT_NAME_LOWER)
+project(minetest)
+
+# Put capitalized PROJECT_NAME into PROJECT_NAME_CAPITALIZED
+string(SUBSTRING ${PROJECT_NAME} 0 1 TMP1)
+string(TOUPPER ${TMP1} TMP1)
+string(SUBSTRING ${PROJECT_NAME} 1 -1 TMP2)
+string(CONCAT PROJECT_NAME_CAPITALIZED ${TMP1} ${TMP2})
```

Comment:
This is stupid. Just `set(PROJECT_NAME_CAPITALIZED "Minetest")`. Forks can change both versions if they want to.